ipaserver/install/ipa_migrate.py

```diff
@@ -0,0 +1,1821 @@
+        self.log_info(f" - Remote Domain:           {self.remote_domain}")
+        self.log_info(f" - Local Host:              {self.local_hostname}")
+        self.log_info(f" - Local Domain:            {self.local_domain}")
+        self.log_info(f" - Remote Suffix:           {self.remote_suffix}")
+        self.log_info(f" - Local Suffix:            {self.local_suffix}")
+        self.log_info(f" - Remote Realm:            {self.remote_realm}")
+        self.log_info(f" - Local Realm:             {self.realm}")
+        if self.args.force is not False:
+            self.log_info(" - Ignored Errors:          "
+                          f"{stats['ignored_errors']}")
+
+        # Schema
+        total_schema = stats['schema_attrs_added'] + stats['schema_oc_added']
+
+        title = ('\nSchema Migration (migrated '
+                 f"{total_schema}/{stats['schema_processed']} definitions)")
+        self.log_info(title)
+        self.log_info('-' * (len(title) - 1))
+        self.log_info(" - Attributes:              "
+                      f"{stats['schema_attrs_added']}")
+        self.log_info(" - Objectclasses:           "
+                      f"{stats['schema_oc_added']}")
+
+        # Configuration
+        title = ('\nDS Configuration Migration (migrated '
+                 f"{stats['config_migrated']}/"
+                 f"{stats['config_processed']} entries)")
+        self.log_info(title)
+        self.log_info('-' * (len(title) - 1))
+        logged_something = self.log_stats(DS_CONFIG)
+        if self.args.verbose or NIS_PLUGIN['count'] > 0:
+            self.log_info(f" - NIS Server Plugin:       {NIS_PLUGIN['count']}")
+            logged_something = True
+        if not self.log_stats(DS_INDEXES) and not logged_something:
+            self.log_info(" - No updates")
+
+        # Database
+        title = ("\nDatabase Migration (migrated "
+                 f"{stats['total_db_migrated']}/"
+                 f"{stats['total_db_entries']} entries)")
+        self.log_info(title)
+        self.log_info('-' * (len(title) - 1))
+        logged_something = False
+        if self.args.verbose or self.args.reset_range:
+            logged_something = self.log_info(
+                f" - DNA Range Resets:        {stats['reset_range']}")
+        if not self.log_stats(DB_OBJECTS) and not logged_something:
+            self.log_info(" - No updates")
+
+        # Display any followup notes
+        title = (f"\nAction Items ({len(self.post_notes)} items)")
+        self.log_info(title)
+        self.log_info('-' * (len(title) - 1))
+        for note in self.post_notes:
+            self.log_info(' - ' + note)
+
+        # The end of the summary
+        self.log_info('=' * 79)
+
+    def connect_to_remote_ds(self):
+        """
+        Connect to the remote DS and store the conn in the class
+        """
+        ldapuri = f"ldap://{self.args.hostname}"
+        insecure_bind = False
```

Comment:
There isa an option to set where the CA cert is, that will trigger startTLS.  And yes I actually couldn't get it to work and told myself I would come back to it and never did.  So I guess we could change the option to "not use TLS" instead of use "StartTLS"

Reply:
This matches the behavior of migrate-ds.